src/lib/resource.js

```diff
@@ -36,6 +41,12 @@ const getMetadataValues = async (resourceUrl) => {
   const metadataUrl = `${resourceUrl}?_format=json`;
   const res = await axios.get(metadataUrl, { maxRedirects: 3 });
   const metadata = res.data;
+
+  // if meta data is a string its not a json object.
+  if (typeof metadata === 'string') {
+    return false;

```

Comment:
![Screenshot 2023-07-31 at 2 30 55 PM](https://github.com/HHS/Head-Start-TTADP/assets/3288586/8c703eef-52a0-4173-8664-074e015a5f59)

I am still seeing many errors around attempts to insert non-JSON data into the metadata column

Reply:
I think I used the same resource as you got this: 

![image](https://github.com/HHS/Head-Start-TTADP/assets/84350609/34023cd8-22ab-4194-903d-39023d4af08f)

https://eclkc.ohs.acf.hhs.gov/sites/default/files/pdf/what-planned-language-approach-eng.pdf